test/e2e/arktos/multi_tenancy/test_suites/crd/test_system_share_crd.yaml

```diff
@@ -0,0 +1,160 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ System Forced Sharing CRD Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies the system forced sharing CRD works in Arktos
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  first_tenant: random_8
+  second_tenant: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${first_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${first_tenant} created\n"
+
+  - Command: ${setup_client_script} ${first_tenant} admin
+
+  - Command: ${kubectl} create tenant ${second_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${second_tenant} created\n"
+
+  - Command: ${setup_client_script} ${second_tenant} admin
+
+########################################################################################
+# Verifying system-share CRD is visible to all tenants
+########################################################################################
+
+  - BeforeTestMessage: Verifying system-share CRD is visible to all tenants ...
+    Command: ${kubectl} apply -f ${test_data_dir}/forced_share_crd.yaml --tenant system
+    OutputShouldContain:
+    - "customresourcedefinition.apiextensions.k8s.io/mustshareknowledges.learning.wisdom.com"
+
+  - Command: ${kubectl} get crds --context ${first_tenant}-admin-context
+    OutputShouldContain: 
+    - "mustshareknowledges.learning.wisdom.com"
```

Comment:
repeated string. possible to put them in a variable?

Reply:
Per offline discussion, we skip the issue of hard-coded values for now.

 These values are defined in the external files. We will try to find a better way to coordinate the external files with the test data in the test suite files .